packages/renderers-rust/public/templates/instructionsCpiPage.njk

```diff
@@ -150,9 +150,9 @@ impl<'a, 'b> {{ instruction.name | pascalCase }}Cpi<'a, 'b> {
           is_writable: remaining_account.2,
       })
     });
-    let {{ 'mut ' if hasArgs }}data = {{ instruction.name | pascalCase }}InstructionData::new().try_to_vec().unwrap();
+    let {{ 'mut ' if hasArgs }}data = borsh::to_vec(&{{ instruction.name | pascalCase }}InstructionData::new()).unwrap();
     {% if hasArgs %}
-      let mut args = self.__args.try_to_vec().unwrap();
+      let mut args = borsh::to_vec(&self.__args).unwrap();
```

Comment:
What's the rational for bumping the borsh crates here? IIRC, we went for the lowest possible version to avoid dependency hell as they are many libraries out there using different version of borsh. Cc @febo that might have a better memory than me here.

Reply:
I was getting build errors about BorshDeserialize not being implemented

Reply:
switching to borsh 0.10